### PR TITLE
Fixing SEO crawling issues and adding useful markup

### PR DESCRIPTION
**Pull request by guillermoga88**
I've added missing titles and schema.org markup.

### CODE REVIEW

File: _includes/banner_blog.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   {% assign current = post %}
 {% endif %}
 
-<section class="banner banner-post banner-blog" {% if current.design.bg_color %}style="background: {{current.design.bg_color}}"{% endif %}>
+<section class="banner banner-post banner-blog" {% if current.design.bg_color %}style="background: {{current.design.bg_color}}"{% endif %}itemscope itemtype="http://schema.org/WPHeader">
```
Comment (alejofernandez) [inline _includes/banner_blog.html]: @guillermoga88 I believe a space is needed after the endif block: `{% endif %} itemscope`

File: _includes/entry_image.html [hunk 2]
```diff
@@ -42,10 +42,13 @@
 <a href="{{ site.baseurl }}{{current.url}}"
   style="{{image_bg_color}} {{bg_merge}}"
   class="entry-thumbnail">
-  <div class="holder">
+  <div class="holder" itemprop="image" itemscope itemtype="http://schema.org/ImageObject">
     <img
     style="{{opacity}} {{size}} {{top}} {{left}}"
     src="{% if current.design.image %}{{ current.design.image }}{% else %}{{ current.design.image_default }}{% endif %}"
     alt="{{ current.title }}">
+    <meta itemprop="url" content="{% if current.design.image %}{{ current.design.image }}{% else %}{{ current.design.image_default }}{% endif %}">
```
Comment (alejofernandez) [inline _includes/entry_image.html]: Aren't `meta` tags allowed only inside `head`?
https://www.w3.org/TR/REC-html40/sgml/dtd.html#head.misc

File: _layouts/post.html [hunk 3]
```diff
@@ -11,61 +11,72 @@
...
 
-  </script>
-  <noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
-  <!-- <a href="//disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a> -->
-</div>
+    </script>
+    <noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
+    <!-- <a href="//disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a> -->
+  </div>
+</article>
```
Comment (alejofernandez) [inline _layouts/post.html]: Please add a newline at the end of the file

**Comment by guillermoga88** [comment]
Done @alejofernandez :)
Let me know if you need anything else!

File: _includes/entry_image.html (hunk 2 above)
Comment (guillermoga88) [inline _includes/entry_image.html]: @alejofernandez it's allowed for microdata if the content can't be marked up without the meta tag. 
http://schema.org/docs/gs.html#advanced_missing

Comment (alejofernandez) [inline _includes/entry_image.html]: Cool thanks! It's nice to learn something new!

File: _includes/banner_blog.html (hunk 1 above)
Comment (guillermoga88) [inline _includes/banner_blog.html]: I thought so too :)
But when I tested it, it showed up with an extra space:
`<section class="banner banner-post banner-blog"  itemscope itemtype="http://schema.org/WPHeader">`

File: _layouts/post.html (hunk 3 above)
Comment (guillermoga88) [inline _layouts/post.html]: Done! @alejofernandez

File: _includes/banner_blog.html (hunk 1 above)
Comment (alejofernandez) [inline _includes/banner_blog.html]: Yep, that part works fine, also tested it but when `current.design.bg_color` has value you end up with `style="background: xxxx"itemscope itemtype="http`

Comment (guillermoga88) [inline _includes/banner_blog.html]: No prob! I'll add that extra space

**Comment by alejofernandez** [comment]
This one is ready to merge, will do it later on today

**Pull request merged**